dask/dataframe/groupby.py

```diff
@@ -1012,6 +1101,33 @@ def std(self, ddof=1, split_every=None, split_out=1):
         result = map_partitions(np.sqrt, v, meta=v)
         return result
 
+    @derived_from(pd.DataFrame)
+    def cov(self, ddof=1, split_every=None, split_out=1):
+        levels = _determine_levels(self.index)
```

Comment:
It'd be nice to have a comment somewhere (probably here is best) laying out the overall strategy.

Reply:
👍